utils/config.py

```diff
@@ -3,6 +3,25 @@
 import os
 import json
 
+from core.config import load_bot_config_from_file, generate_bot_config
+
+try:
+    base_path = (Path(os.path.abspath(__file__)) / '..' / '..').resolve()
+    config_path = base_path / 'config_data.json'
+    config_data = load_bot_config_from_file(config_path.__str__()).get_store_json()
+except FileNotFoundError as fe:
+    print(fe)
+    print("BotConfig error: config_data.json file not found, generating new instance")
+    config_data = generate_bot_config({
+            "youtube_announcement_channels": ["announcements"],
+            "vod_count": 0,  # If the file does not exists evaluate getting this value from a DB
+            "yt_text_channel_id": 1132053178838421554,
```

Comment:
I don't like this hardcoded value :thinking:

Maybe the config should support `null`, or just put a `null` and then error out saying it needs to be configured (if this can't be done through a slash command).

Although, maybe it's fine.

Reply:
I agree, if possible make it a null / None value, bc I don't want it to send it to an unintended channel in the future

Reply:
Also, similar idea with `["announcements"]` then, empty arrays would be a better default?